src/components/Article.jsx

```diff
@@ -18,6 +25,16 @@ export const Article = () => {
       });
   }, [article_id]);
 
+  useEffect(() => {
+    axios
+      .get(
+        `https://news-api-40x5.onrender.com/api/articles/${article_id}/comments`,
+      )
+      .then(({ data: { comments } }) => {
+        setComments(comments);
+      });
+  }, [article_id]);
```

Comment:
Have a think about moving your external api calls into a different utils type file, it would help to clean up you component and would allow multiple components to use those api calls if they needed to! 🚀